CHANGELOG.md

```diff
@@ -9,12 +9,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-- Build of packages for Fedora 34 platform.
-
 ### Changed
 
 ### Fixed
 
+## [1.1.2] - 2022-08-21
+
```

Comment:
Let's also insert a short description of the release here. For example:

https://github.com/tarantool/go-replica/blob/b764ac59f6a7f56af513bf47902b6de72f49a1cb/CHANGELOG.md?plain=1#L58-L62

An in the commit message too.

Reply:
This is often useful to quickly understand the purpose or context of a release.

Reply:
Added brief overview.